wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession+UserSession.swift

```diff
@@ -294,6 +294,30 @@ extension ZMUserSession: UserSession {
         let featureRepository = FeatureRepository(context: syncContext)
         return GetMLSFeatureUseCase(featureRepository: featureRepository)
     }
+
+    @MainActor
+    public func fetchSelfConversationMLSGroupID() async -> MLSGroupID? {
```

Comment:
Actually, now that we have a "use case" `MLSGroupIDProviding`/`MLSGroupIDProvider`, can't this be removed?

Reply:
Will be done in a future refactoring.